src/test/java/nextstep/subway/acceptance/LineAcceptanceTest.java

```diff
@@ -33,6 +99,44 @@ void getLines() {
     @DisplayName("지하철 노선 조회")
     @Test
     void getLine() {
+        // given
+        String 신분당선 = "신분당선";
+        String 색깔 = "bg-red-600";
+        Map<String, String> params = new HashMap<>();
+        params.put("name", 신분당선);
+        params.put("color", 색깔);
+
+        ExtractableResponse<Response> createResponse = RestAssured.given().log().all()
+            .body(params)
+            .contentType(MediaType.APPLICATION_JSON_VALUE)
+            .when()
+            .post("/lines")
+            .then().log().all()
+            .extract()
+        ;
+
+        // when
+        JsonPath jsonPath = createResponse.jsonPath();
+        String createdDate = jsonPath.getString("createdDate");
+        String modifiedDate = jsonPath.getString("modifiedDate");
+        String uri = createResponse.header("Location");
+
+        ExtractableResponse<Response> response = RestAssured
+            .given().log().all()
+            .when().get(uri)
+            .then().log().all()
+                .extract();
+
+        // then
+        assertThat(response.statusCode()).isEqualTo(HttpStatus.OK.value());
+        String name = response.jsonPath().getString("name");
+        String color = response.jsonPath().getString("color");
+        String responseCreatedDate = response.jsonPath().getString("createdDate");
+        String responseModifiedDate = response.jsonPath().getString("modifiedDate");
+        assertThat(name).isEqualTo(신분당선);
+        assertThat(color).isEqualTo(색깔);
+        assertThat(responseCreatedDate).isEqualTo(createdDate);
+        assertThat(responseModifiedDate).isEqualTo(modifiedDate);
```

Comment:
이렇게 긴 검증 부분 역시 모듈화 하거나 코드간의 논리적 유사성을 토대로 세로 밀집도를 구분해줄 필요가 있어요. 
```suggestion
        assertThat(response.statusCode()).isEqualTo(HttpStatus.OK.value());
        
        String name = response.jsonPath().getString("name");
        String color = response.jsonPath().getString("color");
        String responseCreatedDate = response.jsonPath().getString("createdDate");
        String responseModifiedDate = response.jsonPath().getString("modifiedDate");
        
        assertThat(name).isEqualTo(신분당선);
        assertThat(color).isEqualTo(색깔);
        assertThat(responseCreatedDate).isEqualTo(createdDate);
        assertThat(responseModifiedDate).isEqualTo(modifiedDate);
```